### ISSUE TITLE
Browser tab locks up when closing an executing command

### ISSUE THREAD

**Issue by mechevarria** [eclipse-che/che#9764]
![screenshot from 2018-05-21 14-00-29](https://user-images.githubusercontent.com/2657771/40322675-25bc2592-5d01-11e8-9427-2e999fa3cdf4.png)
### Description
When trying to cancel an executed process, the browser tab freezes.  You have to force close the tab and reload the workspace page.

### Reproduction Steps
I created a **node** workspace and imported the following git project

https://github.com/mechevarria/patternfly-ng-seed

I added port **4200/tcp** under servers

In a terminal, I entered the project and did an **npm install**

I then created a run command called **local**

```
cd ${current.project.path}

npm run local
```

under preview I put **${server.4200/tcp}**

Running this command causes no problem, web application loads.

Afterwards I cancel the run process and the tab that the workspace is in locks up. Checking my system, my CPU is at 100% with google chrome as the process (attached)
![screenshot from 2018-05-22 09-00-16](https://user-images.githubusercontent.com/2657771/40363795-95aab790-5d9e-11e8-9777-533307789d38.png)

**OS and version:**    
Fedora 27 amd64

Both Chrome 66 and Firefox 59

INFO: (che cli): Pulling image alpine:3.4
INFO: (che cli): Pulling image eclipse/che-ip:6.5.0
INFO: (che cli): 6.5.0 - using docker 18.05.0-ce / native
INFO: (che config): Generating che configuration...

**Diagnostics:** 

I don't see any errors in the Eclipse Che logs.  I also tested on che 6.4.1 and got the same result.

**Comment by eivantsov** [comment]
@vparfonov IMHO it's worth investigating

**Comment by mechevarria** [comment]
So I was able to get around this issue but upping the memory for the workspace from **2GB** to **8GB**.  No lockup, everything works.  Unfortunately the dev machine logs don't show any out of memory errors so I just tried it as a guess.  Even with 4GB things were a bit unstable.  

Maybe upping the default memory allocated to node.js stack workspaces could be a fix.  The project I am running is an [Angular CLI](https://cli.angular.io/) app that is running a webpack dev server as a node process.  Not sure why it would need so much memory but maybe the node process is generating lots of temp files.

Closing this issue.
